### ISSUE TITLE
ENErrorDomain-Fehler 2

### ISSUE THREAD

**Issue by hansdampf01** [corona-warn-app/cwa-app-ios#823]
![69140163-7E5E-43BF-B967-0BC53D8F442A](https://user-images.githubusercontent.com/55264284/86272980-c4a51900-bbcf-11ea-9b6a-503ca690b9dc.png)

Im getting attached error when opening the cwa app.
Im on iPhone 11 Pro Max 13.5 (17F75) (Jailbroken) and downloaded the App via Appstore. Currently using cwa version 1.0.3 (1) from 01.07.2020.
I got this also with the previous version. 
Since I got this error on my device I thought it only happened to me due to the jailbreak and not using 13.5.1 / 13.6 beta. 
However im also managing my companies devices with Intune in Azure and made the cwa app available for my companies mobile devices. Today a user opened a Ticket for the same issue. The user is on a non-jailbroken iPhone 11 with 13.5.1 and facing the same. 
I already called the hotline shown in the cwa app and opened a ticket about the error. 
Please feel free to contact if you need further details.

**Comment by bmmmm** [comment]
Same error here.
After update.

App reseted via internal function.
As well as uninstalled and reinstalled.

ENErrorDomain-Fehler 13

iPhone XR
iOS 13.5.1
APP-version: 1.0.3 (1)

**PS: 
please write proper change / release note for that amount of money -.-**

**Comment by jwildeboer** [comment]
Error 13 is not error 2 :) Error 11/13 is a known issue and should be fixed with iOS 13.6 as it is in the Apple framework and not in the app.

**Comment by bmmmm** [comment]
@jwildeboer you're right.
thank you! just found the corresponding thread and sorry for spam ;/

I will delete this comment as well, as soon  @jwildeboer deleted his thread ;)

**Comment by camelusferus** [comment]
This might happen also on non-jailbroken devices. I’ve got the same error on an non-jailbroken iPhone X, running iOS 13.5.1, after the update to the CWA v1.0.3. Despite the fact, that the number behind "ENErrorDomain-Fehler" differs, error number 11 and 13 had shown up to me. An deletion and reinstallation of cwa changed nothing.

**Comment by jwildeboer** [comment]
Please be precise. Did you get error 2 or 11/13?

**Comment by camelusferus** [comment]
I get 11/13, not 2.

**Comment by jwildeboer** [comment]
Please add your observations to #759 where error 11/13 is handled. This issue should only be about error 2. thanks!

**Comment by hansdampf01** [comment]
So mal Butter bei die Fische - wann fixt ihr das?
Das ging eine ganze Zeit lang am Anfang, dann habt ihr es kaputt gepatched.
Das Problem wird wohl bei User Enrollment von IOS 13 lieben.

**Comment by SebastianWolf-SAP** [comment]
The original post was on error 2. The new issue is something else - we would appreciate if you report that in a dedicated issue. We've seen such issues only after an iTunes backup restore - did you do something like that?

Moreover, please also report back to us here if you can reproduce error 2 on your device. If not, we would probably close this one as it can't be reproduced anymore.

Mit freundlichen Grüßen/Best regards, 
SW
Corona Warn-App Open Source Team

**Comment by hansdampf01** [comment]
Its happening on all devices which are intune managed enrolled in ios 13 user mode. Every open of the app will produce 2 warning windows about EN Error 2. 

In regards of new error it happens after each update of the app on intune managed devices enrolled in user mode.
I can create a new issue for that.

My best shot is there is an error with ios 13 User Mode enrollment which is a brand new feature that enables you to manage mobile phones gdpr compliant.

**Comment by GPclips** [comment]
Hello @hansdampf01, 

since it has been a while, I kindly like to ask you if this error still occurs?
Could you please give me quick feedback, that we can create an internal tracking id and share this with the dev team and/or Apple?

Also, if I understand you correctly this is happening whenever you open the application, is that correct?

Thanks,
LMM

Corona-Warn-App Open Source Team

**Comment by ndegendogo** [comment]
Although not a precise duplicate, it seems to happen again now. Several users are reporting EN error 2 with current cwa version 1.3.2 after migration from iOS 14.2 beta 2 to beta 3 (#1339).

Could you please clarify with Apple? They should fix it before their final release of 14.2

**Comment by sebastiancomsa** [comment]
I'm also receiving the same error, when I tried to open the app today. I updated my iOS yesterday afternoon to version 14.2 (18B5072f).

Any suggestions? Re-install of the app? Maybe not the best to do and just wait for a fix?

Thanks,

Seb

**Comment by Ein-Tim** [comment]
@sebastiancomsa 
Please see [this](https://github.com/corona-warn-app/cwa-app-ios/issues/1339#issuecomment-709230027) comment from @thomasaugsten, the App doesn't work with iOS 14.2 Beta 3.

**Comment by sebastiancomsa** [comment]
@Ein-Tim i hope this will be fixed soon, I also reported this to Apple via their feedback tool. So hopefully they will also look into this from their side of the Exposure logging code in iOS.

Thank you for the updates though, will patiently wait to have this fixed. Will not expose myself in the meantime! ;)

**Comment by dsarkar** [comment]
Dear @hansdampf01, @sebastiancomsa, community,

We would appreciate feedback on this issue: Any reoccurrence with CWA 1.7.1?

Thanks. Best, DS

---
Corona-Warn-App Open Source Team

**Comment by dsarkar** [comment]
Dear @hansdampf01, @sebastiancomsa, and community,

Thanks for all the contributions here. We will close this issue now. However, should this issue arise again, please re-open it.

Best regards, DS

---
Corona-Warn-App Open Source Team
